projects/1.20/assets/forestry-community-edition/forestry/lang/zh_cn.json

```
@@ -0,0 +1,2360 @@
+  "death.bee.heroic": "%1$s被英雄蜂击杀了！",
+  "effect.forestry.hakuna_matata": "无忧无虑",
+  "effect.forestry.matata": "忧虑袭扰",
+  "allele.forestry.bee_effect.bee_effect_aggressive": "举世皆敌",
+  "allele.forestry.bee_effect.bee_effect_beatific": "皆大欢喜",
+  "allele.forestry.bee_effect.bee_effect_creeper": "苦力怕",
+  "allele.forestry.bee_effect.bee_effect_drunkard": "沉醉",
+  "allele.forestry.bee_effect.bee_effect_exploration": "探索",
+  "allele.forestry.bee_effect.bee_effect_glacial": "冻结",
+  "allele.forestry.bee_effect.bee_effect_fertile": "丰饶",
+  "allele.forestry.bee_effect.bee_effect_heroic": "英雄",
+  "allele.forestry.bee_effect.bee_effect_ignition": "燃烧",
+  "allele.forestry.bee_effect.bee_effect_misanthrope": "末地",
+  "allele.forestry.bee_effect.bee_effect_miasmic": "剧毒",
+  "allele.forestry.bee_effect.bee_effect_mycophilic": "嗜菇",
+  "allele.forestry.bee_effect.bee_effect_none": "无",
+  "allele.forestry.bee_effect.bee_effect_radioactive": "辐射",
+  "allele.forestry.bee_effect.bee_effect_reanimation": "重生",
+  "allele.forestry.bee_effect.bee_effect_repulsion": "排斥",
+  "allele.forestry.bee_effect.bee_effect_resurrection": "复苏",
+  "allele.forestry.bee_effect.bee_effect_snowing": "落雪",
+  "allele.forestry.bee_effect.bee_effect_easter": "复活节",
+  "allele.forestry.bee_effect.bee_effect_sifter": "筛土",
+  "allele.forestry.bee_effect.bee_effect_hakuna_matata": "哈库那马塔塔",
+  "allele.forestry.bee_effect.bee_effect_glow_berry_grow": "辉光",
+  "allele.forestry.bee_effect.bee_effect_phasing": "隧穿",
+  "allele.forestry.bee_effect.bee_effect_ascension": "飞腾",
+  "allele.forestry.activity.activity_diurnal": "昼行",
+  "allele.forestry.activity.activity_nocturnal": "夜行",
+  "allele.forestry.activity.activity_metaturnal": "不睡眠",
+  "allele.forestry.activity.activity_crepuscular": "曙暮",
+  "allele.forestry.activity.activity_cathemeral": "不规律",
+  "allele.forestry.bee_effect.bee_effect_guardian": "守卫者",
+  "allele.forestry.bee_effect.bee_effect_sculk": "幽匿蔓延",
+  "for.beemode.easy.desc": "简化养蜂流程，大幅提升突变概率，完全阻止遗传疲劳。",
+  "for.beemode.hard.desc": "略微降低突变概率，令蜂后的寿命延长50%%。",
+  "for.beemode.hardcore.desc": "突变概率减半，令蜂后的寿命延长500%%，并令其生育能力降低20%%。",
+  "for.beemode.insane.desc": "突变概率变为20%%，令蜜蜂的寿命变为10倍，并令其生育能力降低40%%。",
+  "for.beemode.normal.desc": "推荐一般玩家游玩的模式。",
+  "for.bees.grammar.drone.type": "雄蜂",
+  "for.bees.grammar.drone": "%1$s%2$s",
+  "for.bees.grammar.larvae.type": "幼虫",
+  "for.bees.grammar.larvae": "%1$s%2$s",
+  "for.bees.grammar.princess.type": "公主蜂",
+  "for.bees.grammar.princess": "%1$s%2$s",
+  "for.bees.grammar.queen.type": "蜂后",
+  "for.bees.grammar.queen": "%1$s%2$s",
+  "for.bees.hybrid": "%1$s-%2$s混血",
+  "allele.forestry.bee_species.bee_austere": "苦行",
+  "allele.forestry.bee_species.bee_avenging": "复仇",
+  "allele.forestry.bee_species.bee_boggy": "泥沼",
+  "allele.forestry.bee_species.bee_common": "寻常",
+  "allele.forestry.bee_species.bee_cultivated": "田野",
+  "allele.forestry.bee_species.bee_darkened": "暗黑",
+  "allele.forestry.bee_species.bee_demonic": "恶魔",

```

```suggestion
  "allele.forestry.bee_species.bee_demonic": "魔鬼",
```
正本清源（）
`Devil`和`Demon`是不一样的，但是林业和附属的职业蜜蜂刚好把两个翻译反了